configure.ac

```diff
@@ -114,6 +114,12 @@ AC_ARG_WITH(sim,
 	[with_sim=qemu]
 	)
 
+AC_ARG_WITH(languages,
+        [AS_HELP_STRING([--with-languages=],
+                [Sets the target languages, defaults to c,c++ for bare mental or c,c++,fortran for others])],
+        []
```

Comment:
I'm not familiar enough with `configure/autoconf` etc. but is this way of providing a blank default here, which is overridden in `Makefile.in` and potentially further overridden via `configure --with-languages=...`, the correct way to approach this? Just asking just in case it's not... :-)

Reply:
yes, in order to reach the new ideas - different default values for different stage 2s.